Doc/Compatability-table.md

```diff
@@ -0,0 +1,137 @@
+# Unity types compatibility table
+
+| Module | Type | Can read | Can write | Custom converter
+| --- | --- | --- | --- | --- |
+| AI/NavMesh | _UnityEngine.<i></i>AI_.**NavMeshDataInstance** | ❌<sup>[[2]](#footnote-2)</sup> | ❓<sup>[[3]](#footnote-3)</sup> | ❌<sup>[[2]](#footnote-2)</sup>
+| AI/NavMesh | _UnityEngine.<i></i>AI_.**NavMeshHit** | ✔ | ✔ | ✖<sup>[[1]](#footnote-1)</sup>
+| AI/NavMesh | _UnityEngine.<i></i>AI_.**NavMeshLinkData** | ✔ | ✔ | ✖<sup>[[1]](#footnote-1)</sup>
+| AI/NavMesh | _UnityEngine.<i></i>AI_.**NavMeshLinkInstance** | ❌<sup>[[2]](#footnote-2)</sup> | ❓<sup>[[3]](#footnote-3)</sup> | ❌<sup>[[2]](#footnote-2)</sup>
+| AI/NavMesh | _UnityEngine.<i></i>AI_.**NavMeshQueryFilter** | ✔ | ✔ | ✔
+| AI/NavMesh | _UnityEngine.<i></i>AI_.**NavMeshTriangulation** | ✔ | ✔ | ✔
+| Animation | _UnityEngine_.**Keyframe** | ✔ | ✔ | ✖<sup>[[1]](#footnote-1)</sup>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5461644)